test-framework/common/src/main/java/io/quarkus/test/common/LauncherUtil.java

```diff
@@ -94,8 +95,16 @@ private static void ensureProcessIsAlive(Process quarkusProcess) {
                     "Interrupted while waiting to determine the status of process '" + quarkusProcess.pid() + "'.");
         }
         if (!quarkusProcess.isAlive()) {
-            throw new RuntimeException("Unable to successfully launch process '" + quarkusProcess.pid() + "'. Exit code is: '"
-                    + quarkusProcess.exitValue() + "'.");
+            int exit = quarkusProcess.exitValue();
+            String message = "Unable to successfully launch process '" + quarkusProcess.pid() + "'. Exit code is: '"
+                    + exit + "'.";
+            String osName = System.getProperty("os.name", "generic").toLowerCase(Locale.ROOT);
+            boolean isMac = osName.contains("mac") || osName.contains("darwin");
```

Comment:
I think we might have a util class somewhere for this...

Have you checked?

Reply:
Will fix.